stable/vpa/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: vpa
 description: A Helm chart for Kubernetes Vertical Pod Autoscaler
 type: application
-version: 2.4.0
+version: 2.4.1
```

Since this adds new functionality, it should be considered a minor change, not a patch. 

```suggestion
version: 2.5.0
```